pom.xml

```diff
@@ -920,6 +920,7 @@
                     </systemPropertyVariables>
                     <environmentVariables>
                         <CRAFTER_DATA_DIR>target/test-resources/data</CRAFTER_DATA_DIR>
+                        <MARIADB_ROOT_PASSWD>root</MARIADB_ROOT_PASSWD>
```

Comment:
Was this missing before and causing problems?
Also, doesn't Studio now automatically change the password if it's root?

Reply:
The default configuration is to use the env variable so for the test it has to be provided in the config, I don't know if it is changed later but it was preventing the test from running